lib/apr/views/commerce/commerce_order_slack_view.ex

```diff
@@ -14,10 +14,10 @@ defmodule Apr.Views.CommerceOrderSlackView do
   
   def render(subscription, event, routing_key) do
     case {subscription.theme, event["verb"], event["properties"]} do
-      {"fraud", "submitted", %{"mode" => "buy", "items_total_cents" => cents}} when cents >= 3_000_00 ->        
+      {"fraud", "submitted", %{"mode" => "buy", "items_total_cents" => cents}} ->        
```

Comment:
I'm confused about this requirement. I think removing the 3K check will surface _all_ submitted buy now orders in the fraud channel, and I'd assume that's too much for the team to review?

Reply:
__From Kelly:__ hey! yes we do want to remove it since it actually will be less orders to review. The 10k threshold we're proposing with both make offer and buy now orders will replace this

right now we're also reviewing a ton below that threshold that have been blocked by stripe radar

i believe the 3k rule was put in place but that number doesn't really mean anything but the 10k threshold will be more in line with what's required to review even if the order hasn't been blocked by stripe radar! does that make sense?

Reply:
_continued:_
thanks! the idea is that we'll only review things that are blocked by stripe radar or our rules, or orders above the 10k threshold for anti money laundering purposes. the 3k threshold was arbitrary

Reply:
cc @starsirius

Reply:
So if I'm reading the code correctly, we should replace this 3K check with 10K? The 10K (or precisely 9.5K) check below only applies to "offer" orders and we should have the same check here for "buy" orders. Let me know if I actually misunderstood the :elixir: code here.

Reply:
I think I need to spend some more time making sure I actually know whats going on here, because I think a lot of my assumptions when i did this work were off